src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -384,7 +385,21 @@ class SharedLinkSection extends React.Component<Props, State> {
                         <span className="security-indicator-icon-globe">
                             <IconGlobe height={12} width={12} />
                         </span>
-                        {permissionLevel === CAN_EDIT ? (
+                        // TODO: temporary change to support Canvas not being truly public
+                        {/* eslint-disable-next-line no-nested-ternary */}
```

Comment:
any thoughts if this can be written without needing to do a `eslint-disable`?

Reply:
Used a different approach